application/applicationextension.inc.php

```diff
@@ -2269,4 +2269,46 @@ public function InitStats();
      * @return mixed
      */
     public function LogOperation($oKpiLogData);
+}
+
+/**
+ * Interface to provide messages to be displayed in the "Welcome Popup"
+ *
+ * @api
+ * @private
+ * @since 3.1.0
+ */
+interface iWelcomePopup
+{
+	// Importance for ordering messages
+	// Just two levels since less important messages have nothing to do in the welcome popup
+	const IMPORTANCE_CRITICAL = 0;
+	const IMPORTANCE_HIGH = 1;
```

Comment:
```suggestion
	public const IMPORTANCE_CRITICAL = 0;
	public const IMPORTANCE_HIGH = 1;
```

PHP 7.4 min so we can set visibility